recipes/checkm-genome/post-link.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
```

Comment:
@jj-umn can you please also add an unlnk script?

Reply:
@bgruening I'll look into that.  Thanks.